src/pages/CameraPage.js

```diff
@@ -38,13 +38,24 @@ export default function CameraPage({ onNavigate, handleMemorySubmit }) {
             </>
           )}
         </ImageSection>
-        <InputText
-          aria-label="Choose a name"
-          id="upload-img"
-          type="text"
-          name="title"
-          placeholder="Choose a name"
-          autoComplete="off"
+        <TitleDate>
+          <InputTitle
+            aria-label="Choose a name"
+            id="upload-img"
+            type="text"
+            name="title"
+            placeholder="Choose a name"
+            autoComplete="off"
+            maxlength="30"
+          />
```

Comment:
Where is your `label` element?

Reply:
Jerry told me I can use the aria-label instead.